app/geo/gui/src/ts/twxplore/gui/geo/components/filterComponent/FilterComponent.tsx

```diff
@@ -192,8 +192,8 @@ const FilterComponentImpl: React.FunctionComponent<{featureType: string}> = ({
             );
             return <React.Fragment key={attributeName} />;
           }
-          //If filters have been set
-          case MapFeatureTypeState.FILTERS_SET: {
+          //If filters have been initialized and all work done for setting up a feature type has completed.
+          case MapFeatureTypeState.FINISHED_SETUP: {
             return returnFilterComponent(
```

Comment:
Functions that take more than one parameter should use keyword arguments only e.g.,

```
const myFunc = (kwds: {x: string, y: string}) => {
  const {x, y} = kwds;
...
}

myFunc({x: "x", y: "y"})
```